c2corg_ui/static/menu-discourse/README.md

```diff
@@ -0,0 +1,35 @@
+
+### Menu on the Discourse Forum
+
+The goal was to rebuild the exact same menu in both UI and on the Discourse Forum.
+Only a few icons change (to differenciate between the already-too-similar-ones on Discourse) and the HTML for the menu on Discourse had to be adapted.
+
+If you make any changes to the `sidemenu.html` or `sidemenu.less`, make sure you update the Discourse's menu too.
+
+The HTML changes will have to be adapted manually.
+
+The less file (`sidemenu.less`) was written to suit both the UI and Discourse, even though we won't use the exact same lines in both cases, but the code stays mostly the same.
+
+The files in Discourse should be changed under **the hamburger button** -> **admin** -> **customize (personnaliser)** -> **CSS/HTML** -> and choose from the left list **discourse menu** or create a new style.
+
+
+## LESS -> CSS
+In order to compile the usual sidemenu.less file into pure CSS, use the command:
+
+`./node_modules/.bin/lessc less/sidemenu.less  > sidemenu.css`
```

Comment:
you mean you prefer to add an additional activity to `make less` that will output this? 
I preferred to use this one instead of make less -> was muuuch faster.